paper/paper.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Pynxtools: A framework for generating NeXus files from raw file formats.'
```

Comment:
What about the following terms and thoughts in the title
- pynxtools, I agree,
- NeXus, I agree
- "across disciplines" is one of our USP
- semantic harmonization is another of our USP
- "raw" file formats, I disagree can be used but the term is ambiguous, most file formats store not only raw data

Reply:
pynxtools : A Software Toolbox for Generating and Working wirh NeXus Files